src/open_samus_returns_rando/misc_patches/text_patches.py

```diff
@@ -10,7 +10,7 @@
 # may want to edit all the localization files?
 ALL_TEXT_FILES = {
     # "eu_dutch.txt",
-    # "eu_english.txt",
+    "eu_english.txt",
```

Comment:
That was required for me to work on PAL version. Please check on NTSC.

Reply:
![image](https://github.com/randovania/open-samus-returns-rando/assets/38679103/38b13e66-d52c-4bc6-870f-bc28513610ed)